InWorldz/InWorldz.Phlox.Engine/LSLSystemAPI.cs

```diff
@@ -7700,7 +7699,7 @@ public LSL_List llParseString2List(string str, LSL_List separators, LSL_List in_
                     str = str.Substring(cindex + cdeli.Length);
                 }
             } while (dfound);
-            if (str != "")
+            if (!String.IsNullOrEmpty(str))
```

Comment:
No semantic change: str cannot be null at this point without having caused a null pointer exception.